pipeline/load.py

```diff
@@ -73,7 +72,7 @@ def insert_into_botanist_table(connection: Connection, plant_data: pd.DataFrame)
                 query, {"name": name, "email": email, "telephone": telephone})
 
 
-def insert_into_plant_table(conn: Connection, plant_data: list[dict]) -> None:
+def insert_into_plant_table(connection: Connection, plant_data: list[dict]) -> None:
```

Comment:
Nice renaming of connection to ensure we are using the connection passed in and not the one created in the main